src/components/Common/Combobox.tsx

```diff
@@ -160,7 +160,7 @@ export const Combobox = ({
           (displayedItems?.length > 0 ? (
             displayedItems.map((item, index) => (
               <li
-                className={`flex cursor-pointer flex-col px-3 py-2 ${
+                className={`group flex cursor-pointer flex-col px-3 py-2 ${
```

Comment:
not sure that adding group here is good solution, because every dropdown will have it now
Maybe it's better to add for item which passed to dropdown?

Reply:
in this case group will work only when hovering on the model name (as it works now), not on the list item and the icon will lose background

Reply:
`ShareIcon` highligting doesn't work without `group-hover`